sphinx/source/docs/user_guide/embed.rst

```diff
@@ -124,7 +123,7 @@ Then the corresponding code on the page might look like:
         .then(function(item) { return Bokeh.embed.embed_item(item) })
     </script>
 
-or with modern syntax:
+Or, with modern syntax, like that:
```

Comment:
I think this should also read "like this" (a "that" does not make sense to me here)